sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/CheckAnalysis.scala

```diff
@@ -171,6 +171,7 @@ trait CheckAnalysis extends PredicateHelper {
                         s"appear in the arguments of an aggregate function.")
                   }
                 }
+              case _: PythonUDF => // OK
```

Comment:
I'm not sure why `PythonUDF` is ok? Can you elaborate?

Reply:
Ah I was gonna add the check but forgot, my bad. Let me add proper check for PythonUDF.

Reply:
Fixed.